Physics/CdrPlasma/PlasmaModels/CdrPlasmaJSON/CD_CdrPlasmaJSON.H

```diff
@@ -875,7 +916,30 @@ namespace Physics {
 	@param[in] a_reactionIndex Reaction index (this is the array index in the dielectric reaction array supplied in the JSON file)
 	@param[in] a_reactionJSON  JSON entry for the reaction in the JSON input file.
       */
-      virtual void parseDielectricReactionScaling(const int a_reactionIndex, const json& a_reactionJSON);      
+      virtual void parseDielectricReactionScaling(const int a_reactionIndex, const json& a_reactionJSON);
+
+      /*!
+	@brief Parse secondary emission on domain
+      */
+      virtual void parseDomainReactions();
+
+      /*!
+	@brief Parse reaction rate for domain reactions. 
+	@param[in] a_reactionIndex  Reaction index. 
+	@param[in] a_reactionJSON   Input reaction. Must be one of the entries in the 'electrode reactions' array.
+	@param[in] a_sides          List of the domain sides that this occurs at
+      */
+      virtual void parseDomainReactionRate(const int a_reactionIndex, const json& a_reactionJSON, const std::vector<std::string>& a_sides);      
+
+      /*!
+	@brief Parse domain reaction scaling for a specific reaction.
+	@details This currently exists without much functionality -- it just returns one for all reactions. As we
```

Comment:
Not true..?

Reply:
I agree. I'll remove it for electric and dielectric as well